### DISCUSSION TITLE
Can one instantiate an Ormar model using the .construct() method?

### DISCUSSION THREAD

**Discussion by franzwollang** [collerek/ormar#317] [Q&A] [answered]
In Pydantic, one can use the .construct() method to instantiate a model while skipping validation. I tried it out with an Ormar model and it seemed to work, except there were some random exceptions when I tried to .save() it that gave me the impression that perhaps doing this broke some of Ormar's internals.

Does Ormar work with the .construct() method? Or is there an Ormar equivalent?

**Comment by collerek** [comment, marked as answer]
In theory you probably can but that won't populate relations...

Somehow skipped those method in pydantic 😅🤷 
Skipping the validation can sure come in handy so can you please open an enhancement issue for this?

**Reply by franzwollang** [reply]
Yes, I think that was related to the error I saw. I don't recall exactly it was a day or two ago.

Ok, great! I spawned the issue from this discussion but wasn't able to tag it with the appropriate tag.
